CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 
 This release contains contributions from (in alphabetical order):
 
+Spencer Churchill
+
```

Comment:
@splch, it might be nice to highlight the error mitigation support in the changelog

Reply:
@trbromley, combined the two kwargs into a little blurb and moved to "new features", added a link to an IonQ guide for more info